ext/opentelemetry-ext-opencensusexporter/src/opentelemetry/ext/opencensusexporter/metrics_exporter/__init__.py

```diff
@@ -71,13 +70,7 @@ def export(
         self, metric_records: Sequence[MetricRecord]
     ) -> MetricsExportResult:
         try:
-            responses = self.client.Export(
-                self.generate_metrics_requests(metric_records)
-            )
-
-            # Read response
```

Comment:
I thought this looping through the responses was necessary for grpc to work here, is that not the case?

Reply:
Added that code back.